ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/network_services_point_to_point.yml

```diff
@@ -0,0 +1,88 @@
+---
+# Testing Network Services point-to-point services
+type: pe
+pe:
+  defaults:
+    loopback_ipv4_pool: 192.168.255.0/24
+    vtep_loopback_ipv4_pool: 192.168.254.0/24
+    virtual_router_mac_address: 00:dc:00:00:00:0a
+  nodes:
+    - name: network_services_point_to_point
+      id: 101
+      bgp_as: 101
+      filter:
+        tenants:
+          - TENANT_A
+          - TENANT_B
+
+tenants:
+  - name: TENANT_A
+    mac_vrf_id_base: 10000
+    pseudowire_rt_base: 1000
+    point_to_point_services:
+      # Port-based vpws service
+      - name: TEN_A_ELINE_PORT_BASED
+        type: vpws-pseudowire
+        endpoints:
+          - id: 26
+            nodes: [network_services_point_to_point]
+            interfaces: [Ethernet3]
+          - id: 57
+            nodes: [fake_node]
+            interfaces: [Ethernet3]
+        lldp_disable: true
+      # Sub-interface point-to-point with explicitly defined parent interface (cf next entry)
+      - name: TEN_A_ELINE_VLAN_BASE_EXPLICIT_CHILD
+        type: vpws-pseudowire
+        subinterfaces:
+          - number: 1000
+          - number: 1001
+        endpoints:
+          # ESI dual-homed port-channel endpoint with subinterfaces.
+          - id: 30000
+            nodes: [network_services_point_to_point, fake_peer]
+            interfaces: [Ethernet6, Ethernet6]
+            port_channel:
+              mode: active
+              short_esi: "0102:DEAD:BEEF"
```

Comment:
Let's discuss this again. After seeing the test, it feels wrong to silently ignore this conflicting info.

Reply:
I guess changing this would be a breaking change...

Reply:
```suggestion
            port_channel:
              mode: active
              # This is ignored because the parent is explicitly defined
              # TODO: AVD 6.0.0 decide if it makes sense
              short_esi: "0102:DEAD:BEEF"
```